lib/casbin/effect/allow_and_deny_effector.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'casbin/effect/effector'
+
+module Casbin
+  module Effect
+    class AllowAndDenyEffector < Casbin::Effect::Effector
+      # returns a intermediate effect based on the matched effects of the enforcer
+      def intermediate_effect(effects)
+        return Casbin::Effect::Effector::DENY if effects.include?(Casbin::Effect::Effector::DENY)
+
+        Casbin::Effect::Effector::INDETERMINATE
```

Comment:
Can we use just `INDETERMINATE`? It seems that this constant is declared in the parent class.